src/version.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module ckeditor5/version
+ */
+
+/* global window, global */
+
+import CKEditorError from '@ckeditor/ckeditor5-utils/src/ckeditorerror';
+import { version } from '../package.json';
+
+const windowOrGlobal = window || global;
```

Comment:
How's this supposed to work?

![image](https://user-images.githubusercontent.com/156149/42381561-c0989d80-8131-11e8-9609-ee7a1ea2555c.png)

I think that you need to use `typeof window`.